docs/references.rst

```diff
@@ -0,0 +1,166 @@
+
+
+--------------------
+
+
+:guilabel:`NodeUtil`
+====================
+
+- **Inherits:** `Object <https://docs.godotengine.org/en/stable/classes/class_object.html>`_
+- **Inherited by:** None
+
++------------+-------------------------+
+| Attributes | Methods                 |
++============+=========================+
+||           || :term:`play_animation` |
+||           || :term:`reparent`       |
++------------+-------------------------+
+
+
+.. topic:: Methods
+
+    .. glossary::
+        play_animation
+            **Parameters**
+
+            - AnimationPlayer\_ (`AnimationPlayer <https://docs.godotengine.org/en/stable/classes/class_animationplayer.html>`_
+              or `AnimatedSprite <https://docs.godotengine.org/en/stable/classes/class_animatedsprite.html>`_)
+            - animation (`string <https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_basics.html#string>`_) -
+              The name of the animation.
+            - replay (`bool <https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_basics.html#bool>`_) -
+              If set to true, the current animation will be replayed if it is equal to the animation parameter.
+
+            **Returns**
+
+            void
+
+        reparent
+            **Parameters**
+
+            - child (inherits `Node <https://docs.godotengine.org/en/stable/classes/class_node.html>`_)
+            - new_parent (inherits `Node <https://docs.godotengine.org/en/stable/classes/class_node.html>`_)
+
+            **Returns**
+
+            void
+
+
+--------------------
+
+
+:guilabel:`MathUtil`
+====================
+
+- **Inherits:** `Reference <https://docs.godotengine.org/en/stable/classes/class_reference.html>`_
+- **Inherited by:** None
+
++------------+----------------------------------+
+| Attributes | Methods                          |
++============+==================================+
+|            | :term:`calculate_pivot_rotation` |
++------------+----------------------------------+
+
+
+.. topic:: Methods
+
+    .. glossary::
+        calculate_pivot_rotation
+            Calculates the Pivot rotation such that Origin's look vector points at Target's global_position.
+
+            **Parameters**
+
+            - Pivot (`Position2D <https://docs.godotengine.org/en/stable/classes/class_position2d.html>`_) –
+              The pivot and the object (as a child of the pivot) whose rotation is calculated.
+            - Target (`Position2D <https://docs.godotengine.org/en/stable/classes/class_position2d.html>`_) –
+              The target that the Origin wants to point at.
+            - Origin (`Position2D <https://docs.godotengine.org/en/stable/classes/class_position2d.html>`_) –
+              The origin of the object whose rotation is calculated.
+
+            **Returns**
+
+            The Pivot's rotation angle in radian.
```

Comment:
I think you can omit the "angle" ._. and "rotation" is in radian by default so maybe dun need to specify that?